modules/client-handler/src/integrationTest/java/org/apache/ignite/client/handler/ItClientHandlerMetricsTest.java

```diff
@@ -147,7 +147,7 @@ void testBytesSentReceived(TestInfo testInfo) throws Exception {
         ItClientHandlerTestUtils.connectAndHandshake(serverModule, false, true);
 
         assertTrue(
-                IgniteTestUtils.waitForCondition(() -> testServer.metrics().bytesSent() == 216, 1000),
+                IgniteTestUtils.waitForCondition(() -> testServer.metrics().bytesSent() == 158, 1000),
```

Comment:
This particular line of code checks the size of an exception transferred from the server node. After the changes proposed in this PR, the size (in bytes) was reduced just because the exception message is not extended by the error code and trace identifier.